db/mysql/migrate/000001_initial_schema.up.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE IF NOT EXISTS Service(ID varchar(36) PRIMARY KEY, Name VARCHAR(255));
```

Comment:
Let's add SQL linter. I suggest https://github.com/sqlfluff/sqlfluff but if you'd like an alternative, please let me know.

Reply:
Will look into that

Reply:
Checked other linters, sqlfluff seems superior in all aspects:

- multiple dialects
- configurable
- vscode extension
- most stars compared to other linters, and actively mainained